package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "ravendb-node",
+  "version": "0.0.1",
+  "description": "RavenDB client for nodeJS",
+  "files": [
+    "lib/"
+  ],
+  "main": "lib/ravendb-node.js",
+  "typings": "lib/ravendb-node.d.ts",
+  "scripts": {
+    "_prepublish": "npm run bundle",
+    "quick-build": "tsc --pretty",
+    "bundle": "node scripts/build-bundle.js",
+    "pretest": "node scripts/build-tests.js",
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@git.devs.domerp.com:frontend-projects/ravendb-node.git"
+  },
+  "keywords": [
+    "ravendb",
+    "typescript",
+    "node",
+    "nodejs",
+    "nosql"
+  ],
+  "author": "Hybernating Rhinos",
+  "license": "ISC",
```

Comment:
Please set the `license` field to [match the one set on the repository](https://github.com/ravendb/ravendb-nodejs-client/blob/v4.0/LICENSE).